docs/source/models.rst

```diff
@@ -247,12 +247,34 @@ MLflow provides tools for deploying models on a local machine and to several pro
 Not all deployment methods are available for all model flavors. Deployment is supported for the
 Python Function format and all compatible formats.
 
+.. _pyfunc_deployment:
+
 Deploy a ``python_function`` model as a local REST API endpoint
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 MLflow can deploy models locally as local REST API endpoints or to directly score CSV files.
 This functionality is a convenient way of testing models before deploying to a remote model server.
 You deploy the Python Function flavor locally using the CLI interface to the :py:mod:`mlflow.pyfunc` module.
+The local REST API server accepts the following data formats as inputs:
+
+  * JSON-serialized Pandas DataFrames in the `split` orientation. For example,
+    `data = pandas_df.to_json(orient='split')`. This format is specified using a `Content-Type`
+    request header value of `application/json; format=pandas-split`. Starting in MLflow 0.9.0,
+    this will be the default format if `Content-Type` is `application/json` (i.e, with no format
+    specification).
+
+  * JSON-serialized Pandas DataFrames in the `records` orientation. **THIS FORMAT IS NOT
```

Comment:
I don't think `**` is converting things to bold, maybe just one *? (Like, it's turning it on and off?)

Reply:
That said, maybe one of italics, caps, or bold would be fine here. Don't need to make it so distinctive on the page. Maybe just _This format is not recommended, as it does not preserve column ordering._